utils_cv/detection/plot.py

```diff
@@ -268,6 +268,9 @@ def plot_detections(
     idx: int = None,
     keypoint_meta: Dict = None,
     ax: plt.axes = None,
+    text_size: int = None,
```

Comment:
Instead of the 3 variables, should this take a PlotSettings() object as input?

Reply:
The function overrides plot settings multiple places with hard coded settings (separate colors for GT vs. detection, separate colors for keypoint vs boxes). These are the only default settings. These are the default settings that are relevant for smaller/larger image annotations